extensions/android-paging/build.gradle

```diff
@@ -6,11 +6,6 @@ archivesBaseName = 'sqldelight-android-paging'
 android {
   compileSdkVersion versions.compileSdk
 
-  compileOptions {
-    sourceCompatibility JavaVersion.VERSION_1_7
-    targetCompatibility JavaVersion.VERSION_1_7
-  }
```

Comment:
Not sure if there's a reason why we're using Java 7 here, but if there is, I can revert the changes (and perhaps add a comment as to why we've got this block). Same goes with `extensions/android-paging3/build.gradle` below.